### ISSUE TITLE
homing_override and safe_z_homing cannot be used simultaneously

### ISSUE THREAD

**Issue by youziStar** [3DPrintDemon/Demon_Klipper_Essentials_Unified#81]
After my content includes the following:
![image](https://github.com/user-attachments/assets/bd0463bb-905c-4d0b-8fe5-dc7516697518)
![image](https://github.com/user-attachments/assets/afed8bca-5a68-4fa8-b521-691e5f2e8e6d)
I used Cartographer Probe，I suspect it's the problem。
Below is my print configuration, everything else should be default.
[printer.txt](https://github.com/user-attachments/files/18262337/printer.txt)

**Comment by 3DPrintDemon** [comment]
Yes the problem is you have safe homing & homing override enabled. The macros have their own homing override system, so you’ll need to disable your safe homing section.

**Comment by Tofusang** [comment]
I m using IDM as well, comment out your safe_z_home section will do the job. But I have questions of how to better integrate IDM with DemonKlipper macros.
